src/cmap/auth/mongodb_aws.ts

```diff
@@ -253,61 +251,3 @@ function deriveRegion(host: string) {
 
   return parts[1];
 }
-
```

Comment:
Moving this all out of the AWS module into utils as Azure will reuse this as well as GCP.